docs/UserGuide.md

```diff
@@ -21,7 +21,40 @@ Harmonia is a **desktop app for managing tasks related to your academic life, op
 - [Command Summary](#command-summary)
 
 
-## Quick start
+## 1. About
+
+### 1.1 Purpose
+This user guide aims to teach you how to use Harmonia to systematically organize your student life. It walks you through all the commands Harmonia has and examples on how to use them. By the end of the guide, you should have a better understanding on how to use Harmonia to help you organise your life.
+
+### 1.2 How to use this guide 
+This guide is designed to be read from top to bottom. At the same time, this guide provides ease of navigation, where you can quickly access the segment you are looking for. To this end, the table of contents summarizes all the different sections of our user guide, and it links you to the section of the guide which you wish to look at in detail. After each section, there is also a “return to top” link which brings you back to the table of contents quickly.
+
+### 1.3 Special Symbols
+This user guide denotes information that does not flow as part of the text using special symbols enclosed in a box.
+
+#### 1.3.1 Note
+Instructions or pieces of information that occur in an information box are additional information that can be useful to you.
+
+<div markdown="span" class="alert alert-primary">:bulb: **Note:** This is an example note.</div>
+
+#### 1.3.2 Warning
+Instructions that occur in a warning box you are advised to heed, or else unintended consequences may occur.
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Warning:** This is an example warning</div>
+
+### 1.4 Graphical User Interface (GUI)
+Figure 1.4 depicts the user interface of Harmonia. The following descriptions explain the interface from top to bottom. 
+1. At the top of Harmonia is the toolbar, where you can access the `File` and `Help` options. 
+2. Below the toolbar is the **command box** where you can input your commands into. This will be the main way in which you will interact with Harmonia. 
+3. The result of the command will be output into the **command result box** beneath it. This will be especially useful in helping you understand whether your command has been successfully processed by Harmonia, or if there are any errors. 
+4. The mainframe of Harmonia is the **task list**, where you will be able to see all of your tasks. This frame will update accordingly your commands, and acts as a visual representation of your data. 
```

Comment:
Should `update accordingly your commands` be `update accordingly to your commands`?

Reply:
Ah yes! Thank you for pointing that out! fixed! 👍